iaso/api/mobile/org_units.py

```diff
@@ -244,6 +276,15 @@ def boundingbox(self, request):
         # merge the bbox
         return Response(data={"results": results})
 
+    @action(detail=True, methods=["get"])
+    def reference_instances(self, request, pk=None):
+        """
+        List the reference instances of the given `OrgUnit` pk.
+        """
+        org_unit = self.get_object()
+        serializer = MobileOrgUnitWithReferenceInstancesSerializer(org_unit)
+        return Response(serializer.data)
```

Comment:
The list should be paginated.
Also, I assume the `self.get_object()` will take care of filtering by `app_id`?
Finally, there should be two parameters: `last_sync` and `version_count` which are not handled here, AFAICT

Reply:
> Also, I assume the self.get_object() will take care of filtering by app_id?

Yes sir.